pytest/empty-contract-rs/src/lib.rs

```diff
@@ -1,5 +1,5 @@
-use borsh::{BorshDeserialize, BorshSerialize};
 use near_sdk::{env, metadata, near_bindgen};
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```

Comment:
```suggestion
static ALLOC: near_sdk::wee_alloc::WeeAlloc = near_sdk::wee_alloc::WeeAlloc::INIT;
```

Reply:
Done in https://github.com/nearprotocol/nearcore/pull/2991/commits/5c07c21f0057a0cf9a1c63810fcc1006daa1f8ac